src/js/containers/Transfer/PostTransfer.js

```diff
@@ -32,24 +31,37 @@ export default class PostTransfer extends React.Component {
     if (Object.keys(this.props.transfer.errors.sourceAmount).length !== 0) {
       return
     }
+    
     if (Object.keys(this.props.transfer.errors.destAddress).length !== 0) {
       return
     }
+  
+    this.props.dispatch(transferActions.setIsValidating(true));
```

Comment:
Why you use props instead of state? Having other components using this props?

Reply:
It should be state as you said